resim/geometry/proto/oriented_box.proto

```diff
@@ -0,0 +1,16 @@
+// Copyright 2023 ReSim, Inc.
+//
+// Use of this source code is governed by an MIT-style
+// license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+syntax = "proto3";
+
+import "resim/transforms/proto/se3.proto";
+
+package resim.geometry.proto;
+
+message OrientedBoxSE3 {
+    resim.transforms.proto.SE3 reference_from_box = 1;
+    repeated double            extents            = 2;
```

Comment:
Not sure how to consistently label units, or if we should.

Reply:
In this case, I went with this to match the fields in geometry::OrientedBox exactly. I think the idea of an oriented box shouldn't necessarily be opinionated about units, but at some level (maybe detections in this case) we'll have to be?